source/reference/api/reverse.txt

```diff
@@ -0,0 +1,90 @@
+.. _c2c-api-reverse:
+
+===========
+``reverse``
+===========
+
+.. include:: /includes/preview-warning
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Description
+-----------
+
+Reverses the direction of a committed sync operation.
+
+For example:
+
+- You have a ``COMMITTED`` sync operation.
+
+- ``cluster0`` is the source and ``cluster1`` is the destination.
+
+- After the sync operation is ``COMMITTED``, new writes occur only on
+  the destination cluster.
```

Comment:
I would add one more statement to clarify:  "The source cluster will now not accept new writes"